src/styles/index.scss

```diff
@@ -1,3 +1,89 @@
+$font-family: Arial, sans-serif;
+$font-size: 30px;
+$cell-color: #eee;
+$cell-size: 100px;
+$border-size: 1px;
+$gap-size: 1px;
+$cell-plus-border-size: $cell-size + $border-size;
+
+* {
+  box-sizing: border-box;
+  font-family: $font-family;
+  font-size: $font-size;
+}
```

Comment:
better use *  only for box-sizing: border-box; or don't use on the whole

and for general font-family and font-size use `body`

```
body {
  font-family: $font-family;
  font-size: $font-size;
}
```